ui/src/components/facets/FacetHeader.js

```diff
@@ -29,27 +29,47 @@ const styles = {
     fontWeight: 600,
     padding: "11px 18px 0 0",
     textAlign: "right"
+  },
+  removeFacet: {
+    padding: "7px 0 0 50px"
   }
 };
 
 class FacetHeader extends Component {
+  state = {
+    hoveredOver: false
+  };
+
   constructor(props) {
     super(props);
+    this.mouseEnter = this.mouseEnter.bind(this);
+    this.mouseLeave = this.mouseLeave.bind(this);
+    this.onClose = this.onClose.bind(this);
```

Comment:
handleClose or handleClick.

Let's pick one (handleClose, handleRemove, etc) and use the same name in App.js and this file.